stestr/commands/init.py

```diff
@@ -38,8 +40,11 @@ def init(repo_type='file', repo_url=None):
         for failures.
     :rtype: int
     """
-
-    util.get_repo_initialise(repo_type, repo_url)
+    try:
+        util.get_repo_initialise(repo_type, repo_url)
+    except FileExistsError:
+        stdout.write('The specified repository already exists\n')
+        return 0
```

Comment:
oh, my bad about mkdir, my 'mkdir' was aliased to 'mkdir -p'... :-p So, it returns 1 when I unaliase it.

I understand what your concern now. Hmm, I'm fine ```return 1``` here. I think it should be warning. And I think the message can be improved like 'The specified repository already exists. You should check the repository and remove it if you really want to initialize.\n' or something like that.